guides/doc-Provisioning_Guide/topics/configuring-the-discovery-service.adoc

```diff
@@ -10,7 +10,7 @@ If you select `discovery` to boot the Discovery image, after a few minutes, the
 
 The Discovery service is enabled by default on {ProjectServer}.
 However, the default setting of the global templates is to boot from the local hard drive.
-To change the default setting, in the {ProjectWebUI}, navigate to *Administer* > *Settings*, and click the *Provisioning* tab.
+To change the default setting, in the {ProjectWebUI}, navigate to *Administer* *>* *Settings*, and click the *Provisioning* tab.
```

Comment:
I'm unsure The IBM style guide says the > should also be bold. Is this the case for Satellite/Foreman?

Reply:
IBM Style Guide definitely applies for Foreman Docs, but I am unsure on why we bother having the `>` bold or not. I'm always using `*a > b*` without any issues.

I recommend we don't change this in this PR (so please revert this change for now). Maybe you can open an issue to discuss this. Afterwards, maybe we can write a script to fix this automatically.

Reply:
I've looked in the IBM style and see this: 
"Use bold for menus, menu items, and separator symbols. If a menu contains variable
menu items, use lowercase bold italic." on page 218. So this is indeed a style change that should be updated at some point, albeit extremely trivial. 
If you are introducing style changes, please note it in the PR so that others can understand why you are doing it. A PR should have a problem summary that the PR is solving. 
The way to do this would not be to add *>* but to enclose the entire menu cascade in the same highlighting * Item > Item 2 > Item 3*.
If you introduce this change here, you are making this section inconsistent with the rest of the guides. 
I would leave this until a later time when all the higher priority work is addressed.